src/charts/createHistogramBoxPlotControls.js

```diff
@@ -18,8 +18,12 @@ export function createHistogramBoxPlotControls(this_, d) {
       .attr('checked', true);
 
     commonScaleCheckbox.on('change', function() {
-      d3select(this_).selectAll('.wc-chart').remove();
```

Comment:
In the future, it would be nice to have the update formatting from prettier in a separate commit, so that the actual changes are easier to identify.

Reply:
Yeah, perhaps just prettier the dev branch that we merge feature branches into.